canonical/ubuntu.com
# Change /advantage to /pro

## Done

- Change /advantage to /pro

## QA
Go to https://ubuntu-com-12038.demos.haus/advantage click around a bit. Make a purchase. You should no longer see /advantage anywhere. Will be tested more with the new product selector anyway.  

## Issue / Card

Fixes #https://github.com/canonical/commercial-squad/issues/718
